web/src/main/java/org/openmrs/web/filter/GZIPRequestStream.java

```diff
@@ -43,5 +44,22 @@ public int read() throws IOException {
 	public int read(byte[] b) throws IOException {
 		return zipInput.read(b);
 	}
+
+	@Override
+	public boolean isFinished() {
```

Comment:
Are these methods part of the upgrade?

Reply:
yes @dkayiwa  they are part of the upgrqade

Reply:
@dkayiwa  were you able to look through this PR, is it ready for merging, or is there  some more work I need to do? Let me know.

Reply:
Hello @dkayiwa is there still something stopping this from being merged?